CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## \[Unreleased\]
 
+### Added
+### Removed
+### Changed
+### Fixed
+
+
+## 2024-02-12: v0.18.0-dev.3
```

Comment:
Please just keep the headings that are populated under a release.

Reply:
makes sense - addressed.